zip-1014.rst

```diff
@@ -224,12 +233,12 @@ The funds SHALL be received and administered by ZF. ZF MUST disburse them as
    for reelection. The Major Grant Review Committee is subject to the same
    conflict of interest policy that governs the ZF Board of Directors (i.e. they
    MUST recuse themselves when voting on proposals where they have a financial
-   interest). At most one person with association with the ECC, and at most one
-   person with association with the ZF, are allowed to sit on the Major Grant
-   Review Committee. "Association" here means: having a financial interest,
-   full-time employment, being an officer, being a director, or having an
-   immediate family relationship with any of the above. The ZF SHALL continue to
-   operate the Community Panel and SHOULD work toward making it more
+   interest). At most one person with association with the BP/ECC, and at most
+   one person with association with the ZF, are allowed to sit on the Major
+   Grant Review Committee. "Association" here means: having a financial
+   interest, full-time employment, being an officer, being a director, or having
+   an immediate family relationship with any of the above. The ZF SHALL continue
+   to operate the Community Panel and SHOULD work toward making it more
    representative and independent (more on that below).
 
```

Comment:
Just below this is the Direct Grant section, which says:
```
It may be deemed better, operationally or legally, if the Major Grant funds
are not accepted and disbursed by ZF, but rather directly assigned to the
grantees. Thus, the following mechanism MAY be used in perpetuity for some
or all grantees, if agreed upon by both ECC and ZF before Network Upgrade 4
(Canopy) activation: [...]
```
Should this be changed to "if agreed upon by both BP/ECC and ZF before
Network Upgrade (Canopy) activation:"?

(This doesn't actually apply to Canopy, but it could apply in future upgrades. Similarly for the "ECC and ZF SHALL implement this list in any implementations of the Zcash consensus rules they maintain." in the immediately following
paragraph.)

Reply:
@str4d pointed out that the parties to the Trademark Agreement are still ECC and ZF. BP is not a party to that agreement. Therefore this should not be changed.